src/audio/audio.cpp

```diff
@@ -558,7 +558,7 @@ void Audio::doCapture()
 {
     QMutexLocker lock(&audioLock);
 
-    if (!alInDev || !inSubscriptions)
+    if (!isInputReady())
```

Comment:
Caution: This will cause a deadlock! There's a reason I privatized some things. Anyway, you could write an `Audio::Private::isInputReady` method.